addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -303,16 +303,99 @@ def account_partial_reconcile(env):
     set_workflow_org = models.BaseModel.step_workflow
     models.BaseModel.step_workflow = lambda *args, **kwargs: None
     cr = env.cr
+
+    # Execute a direct reconciliation for the most common and easiest use
+    # case, that involves reconciling two moves, for the full amount, and in
+    # the company currency. This will dramatically improve the overall
+    # performance of the migration of reconciliations.
+    openupgrade.logged_query(cr, """
```

Comment:
As this query only does a SELECT, in the end it changes nothing in the database.